INSTALL.md

```diff
@@ -1,5 +1,4 @@
 Building Bitcoin
 ================
 
-See doc/build-*.md for instructions on building the various
-elements of the Bitcoin Core reference implementation of Bitcoin.
+See [doc/build-\*.md](/doc) for instructions on building the various elements of the Bitcoin Core reference implementation of Bitcoin.
```

Comment:
```suggestion
See [doc/build-\*.md](/doc).
```

I think the whole file can be replaced by a simple <See link>. Though, that might just be me.